doc/nrf/doc_styleguide.rst

```diff
@@ -57,6 +57,16 @@ It is also possible to define more than one default link text for a link, which
    .. _`Link text one`:
    .. _`Link text two`: http://..
 
+Breathe Domains
+===============
+
+In order to link to Doxygen content from RST, the following domains are to
+be used:
+
+* Function: ``:cpp:func:``
+* Structure: ``:c:type:``
+* Enum: ``:cpp:enumerator:``
+* Macro: ``:c:macro:``
 
```

Comment:
Others I have used (might not be correct though):
:cpp:member:`nrf_esb_payload::noack`
:cpp:class:`event_header`

Reply:
I don't think they should be using the `:cpp:class:` but I can add a comment about the `:member:` one

Reply:
Should the member also be used to link to one object in an enum? Or would that be a different command?